packages/block-library/src/comments-pagination-next/index.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Server-side rendering of the `core/comments-pagination-next` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/comments-pagination-next` block on the server.
+ *
+ * @param array    $attributes Block attributes.
+ * @param string   $content    Block default content.
+ * @param WP_Block $block      Block instance.
+ *
+ * @return string Returns the next posts link for the query pagination.
```

Comment:
A copy and paste nitpicking:
```suggestion
 * @return string Returns the next posts link for the comments pagination.
```

Reply:
Fixed!